app/views/handlers/individual_response.py

```diff
@@ -843,7 +856,19 @@ def handle_get(self):
     def handle_post(self):
         if self.selected_option == self.confirm_option:
             self._update_section_status(CompletionStatus.INDIVIDUAL_RESPONSE_REQUESTED)
-            self._publish_fulfilment_request(self.mobile_number)
+            try:
+                self._publish_fulfilment_request(self.mobile_number)
+            except PublicationFailed:
+                retry_url = url_for(
+                    "individual_response.individual_response_text_message_confirm",
+                    list_item_id=self._list_item_id,
+                    **self._request_args,
+                )
+                return (
+                    render_template("errors/individual_response", retry_url=retry_url),
+                    500,
+                )
```

Comment:
I think we need to decide a consistent approach for rendering error pages on exceptions. In #318 this is handled in `errors.py`.

Reply:
I actually prefer the approach in #318 - it's at the least consistent with what we currently do